### PR TITLE
Add mitigation field into Flaw and update SRT notes generator

### PR DESCRIPTION
**Pull request by jobselko**
Closes OSIDB-584

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Integrate Bugzilla backwards sync into the flaw and affect save (OSIDB-240)
 - Introduce Bugzilla API key as a serializer attribute (OSIDB-368)
 - Implement non-empty source validation (OSIDB-759)
+- Implement mitigation field into Flaw and update SRT notes generator (OSIDB-584)
```
Comment (osoukup) [inline docs/CHANGELOG.md:31]: nitpick (you can ignore): Implement field sounds a bit weird to me :smiley_cat: I would say introduce or add etc.

**Review by osoukup — APPROVED** [review]
LGTM. Good job!

File: docs/CHANGELOG.md (hunk 1 above)
Comment (jobselko) [reply docs/CHANGELOG.md]: Thank you! Updated.

**Comment by JakubFrejlach** [comment]
@jobselko Ah, I started review during the time you've merged. :grin:

**Pull request merged**